packages/marko-web-inquiry/browser/default-form.vue

```diff
@@ -139,9 +151,14 @@
       @verify="onVerify"
       @expired="onExpired"
     />
-    <button type="submit" class="btn btn-primary" :disabled="loading">
+    <button type="submit" class="btn btn-primary form-group" :disabled="loading">
```

Comment:
Why was `form-group` added to the submit button?

Reply:
That was just to add some spacing below the button.  I only chose the form-group class because the rest of the rows use that for spacing, so it was just to keep it spaced consistently, but I'd be happy to do it another way instead? :)